app/controllers/api/automation_requests_controller.rb

```diff
@@ -44,12 +44,12 @@ def deny_resource(type, id, data)
 
     def find_automation_requests(id)
       klass = collection_class(:requests)
-      return klass.find(id) if User.current_user.miq_user_role.request_admin_user?
+      return klass.find(id) if User.current_user.role_allows?(:identifier => "miq_request_approval")
```

Comment:
Perhaps this can be put in a method like you did for the other ones. Something like `MiqUserRole#request_approver_user?` This way if we want to switch this logic to looks at a different feature we wouldn't have to change all these lines.

Reply:
Hmm. Not sure that was the problem in this case.

We have so many features. Not sure if we want to introduce a `*_user?` for every one of them.

Also, in the UI and API tend to use identifiers in most cases.
Not sure if this case in particular is worthy of a special method.

To be honest, I was looking forward to getting rid of all of them except for the super user one

Reply:
I think request admin and report admin are extremely large parts of the UI warranting methods for these types of users.  Regardless, I'd prefer to have too many methods on user role than have so many hardcoded feature identifiers.